elements/map/src/select.ts

```diff
@@ -20,44 +20,189 @@ export type SelectOptions = Omit<
   layer?: EoxLayer;
   style?: import("ol/style/flat.js").FlatStyleLike;
   overlay?: import("ol/Overlay").Options;
+  active?: boolean;
 };
 
-export function addSelect(
-  EOxMap: EOxMap,
-  layerId: string,
-  options: SelectOptions
-) {
-  if (EOxMap.interactions[options.id]) {
-    throw Error(`Interaction with id: ${options.id} already exists.`);
-  }
+export class EoxSelectInteraction {
```

Comment:
Just a minor naming thing, we normally prefix all our classes with `EOx`, so in this case this could be `EOxSelectInteraction` if that's ok with you.

Reply:
sure, I was waffling back and forth between the 2 naming options anyways